lib/utils/status.js

```diff
@@ -46,7 +46,7 @@ function status(uri, options, log, useColor) {
     let openOptions = {};
     let openMessage = 'Unable to open browser';
 
-    if (typeof options.open === 'string') {
+    if (typeof options.open === 'string' || Array.isArray(options.open)) {
```

Comment:
hm, why is array, based on docs https://github.com/sindresorhus/open it should be object

Reply:
Options is an object, which takes `app`. App takes either a string or an array. https://www.npmjs.com/package/open#usage . This keeps with the existing API of this project, passing only the `app` value for `open` in options.

i.e:
```
{
  open: "google chrome"
}
```

and

```
{
  open: ['google chrome', '--incognito']
}
```

Reply:
Developers can want to use other options example: `{wait: true}`, also in future package can accept more arguments

Reply:
Good call @evilebottnawi

Reply:
I should have some time tomorrow night to work on this again